rimapi/serializers.py

```diff
@@ -0,0 +1,28 @@
+from re import search
+from rest_framework import serializers
+from rim.models import Equipment, EquipmentType
+from .models import ApiError
+
+class EquipmentSerializer(serializers.ModelSerializer):
+    CPU = serializers.CharField(max_length=64, required=False, source='cpu')
+    Displays = serializers.JSONField(required=False, source='displays')
+    GPUs = serializers.JSONField(required=False, source='GPU')
+    Hostname = serializers.CharField(max_length=100, required=False, source='hostname')
+    Manufacturer = serializers.CharField(max_length=30, required=False, source='manufacturer')
+    Model = serializers.CharField(max_length=30, source='equipment_model')
+    NICs = serializers.JSONField(required=False, source='network_cards')
+    RAM = serializers.CharField(max_length=10, required=False, source='memory')
```

Comment:
I think you forgot to change the max_length on this to match the new max_length in the Equipment model